### DISCUSSION TITLE
Raw view is not updating

### DISCUSSION THREAD

**Discussion by Ash-IPRANOps** [community/community#47028] [Repositories] [answered]
Bug

I updated the file(s) and after several hours, the raw view will never change or update to match the committed changes that were made several hours ago. This was not a problem before, but it started today or recent. Any Idea?

**Comment by OneCDOnly** [comment, marked as answer]
Please see: https://github.com/community/community/discussions/46758

**Comment by AZYsGithub** [comment]
> Bug
> 
> I updated the file(s) and after several hours, the raw view will never change or update to match the committed changes that were made several hours ago. This was not a problem before, but it started today or recent. Any Idea?

Same, that happended to me while update my code..

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Ash-IPRANOps ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by Ash-IPRANOps** [comment]
Issue resolved, thank you.

**Comment by ghost** [comment]
Trying to even update one word from a typeo takes five minutes for raw imo this website is doing this on purpose to rate limit there users

**Comment by hoolio61** [comment]
I edited the data and recommitted it, and now it works.

![image](https://github.com/user-attachments/assets/363164c3-613b-42d0-89e5-67d336651b41)

**Reply by neoarz** [reply]
it works for me sometimes
